packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -938,6 +939,101 @@ describe("BnsConnection", () => {
     });
   });
 
+  describe("getTx", () => {
+    it("can get a transaction by ID", async () => {
+      pendingWithoutBnsd();
+      const connection = await BnsConnection.establish(bnsdTendermintUrl);
+
+      // by non-existing ID
+      {
+        const nonExistentId = "abcd" as TransactionId;
+        await connection
+          .getTx(nonExistentId)
+          .then(fail.bind(null, "should not resolve"), error =>
+            expect(error).toMatch(/transaction does not exist/i),
+          );
+      }
+
+      // by existing ID (from lisk/init.sh)
```

Comment:
lisk/init.sh in BNS tests?

Reply:
Whoops, leftover from copy/pasting the test structure.